index.js

```diff
@@ -1,95 +1,226 @@
+  addHighlighting(homeIcon);
+  removeHighlighting(bookmarkIcon);
+  removeHighlighting(createCardIcon);
+  removeHighlighting(profileIcon);
 
   heading.textContent = "Quiz-App";
 });
 
 bookmarksButton.addEventListener("click", () => {
-  homeSection.classList.add("hidden");
-  bookmarkSection.classList.remove("hidden");
-  createSection.classList.add("hidden");
-  profileSection.classList.add("hidden");
+  hideSection(homeSection);
+  showSection(bookmarkSection);
+  hideSection(createSection);
+  hideSection(profileSection);
 
-  homeIcon.classList.remove("navigation-item--active");
-  bookmarkIcon.classList.add("navigation-item--active");
-  createIcon.classList.remove("navigation-item--active");
-  profileIcon.classList.remove("navigation-item--active");
+  removeHighlighting(homeIcon);
+  addHighlighting(bookmarkIcon);
+  removeHighlighting(createCardIcon);
+  removeHighlighting(profileIcon);
 
   heading.textContent = "Bookmarks";
 });
 
 createButton.addEventListener("click", () => {
-  homeSection.classList.add("hidden");
-  bookmarkSection.classList.add("hidden");
-  createSection.classList.remove("hidden");
-  profileSection.classList.add("hidden");
+  hideSection(homeSection);
+  hideSection(bookmarkSection);
+  showSection(createSection);
+  hideSection(profileSection);
 
-  homeIcon.classList.remove("navigation-item--active");
-  bookmarkIcon.classList.remove("navigation-item--active");
-  createIcon.classList.add("navigation-item--active");
-  profileIcon.classList.remove("navigation-item--active");
+  removeHighlighting(homeIcon);
+  removeHighlighting(bookmarkIcon);
+  addHighlighting(createCardIcon);
+  removeHighlighting(profileIcon);
 
-  heading.textContent = "Create Question";
+  heading.textContent = "Create Card";
 });
 
 profileButton.addEventListener("click", () => {
-  homeSection.classList.add("hidden");
-  bookmarkSection.classList.add("hidden");
-  createSection.classList.add("hidden");
-  profileSection.classList.remove("hidden");
+  hideSection(homeSection);
+  hideSection(bookmarkSection);
+  hideSection(createSection);
+  showSection(profileSection);
 
-  homeIcon.classList.remove("navigation-item--active");
-  bookmarkIcon.classList.remove("navigation-item--active");
-  createIcon.classList.remove("navigation-item--active");
-  profileIcon.classList.add("navigation-item--active");
+  removeHighlighting(homeIcon);
+  removeHighlighting(bookmarkIcon);
+  removeHighlighting(createCardIcon);
+  addHighlighting(profileIcon);
 
   heading.textContent = "Profile";
 });
 
-/* SET NAVIGATION ACTIVE BUTTON */
+function showSection(section) {
+  section.classList.remove("hidden");
+}
+
+function hideSection(section) {
+  section.classList.add("hidden");
+}
+
+function addHighlighting(icon) {
+  icon.classList.add("navigation-item--active");
+}
+
+function removeHighlighting(icon) {
+  icon.classList.remove("navigation-item--active");
+}
+
+/* CREATE CARD FORM */
+const createCardForm = getElement("createCardForm");
```

Comment:
For me this constant sounds like a function: `create a card form`. Please consider a different wording to avoid confusion.